pkg/storaged/overview.jsx

```diff
@@ -24,9 +24,7 @@ import { StoragePlots } from "./plot.jsx";
 
 import { FilesystemsPanel } from "./fsys-panel.jsx";
 import { NFSPanel } from "./nfs-panel.jsx";
-import { MDRaidsPanel } from "./mdraids-panel.jsx";
-import { VGroupsPanel } from "./vgroups-panel.jsx";
-import { VDOsPanel } from "./vdos-panel.jsx";
+import { ThingsPanel } from "./things-panel.jsx";
```

Comment:
StorageDevicesPanel wouldn't have hurt either, but let's let that slip as developer's humor :grin: